packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -100,7 +100,7 @@ export class FileTreeWidget extends TreeWidget {
 
     protected getNodeTooltip(node: TreeNode): string | undefined {
         const uri = UriSelection.getUri(node);
-        return uri ? uri.path.toString() : undefined;
+        return uri ? (uri.scheme === 'ssh' ? uri.toString() : uri.path.toString()) : undefined;
```

Comment:
@Geqi-Yan I'm not sure this is the proper fix, why should the `navigator` know about `ssh` schemes?
Perhaps we should update the following code to use a `labelProvider` and introduce a new label provider for `ssh` schemes.

Reply:
When a node of remote folder is hovered, tooltip text of this node was root path  '/' before. However, I think it is more reasonable to display more information such as the combination of scheme `ssh` and attribute value of authority (`uri.tostring()` returns a formatted uri includes these information). Using scheme `ssh` to distinguish is a little bit hardcoded, and I am not sure if it is correct. A `labelProvider` seems like a good idea.

Reply:
I have verified that this is a great submission, which solves the problem that the file tree cannot be displayed in SSH scenarios.

Reply:
@Geqi-Yan do you have examples of expected SSH URIs to help understand?

Was only one node displaying `/` or were all the nodes doing this?

Reply:
The URI for one of the SSH nodes is shown in the picture below:
![1](https://user-images.githubusercontent.com/74350258/101432959-c7f1bd00-3944-11eb-9938-6622db59847d.png)
All the nodes with `ssh` scheme display `/`. To make things clearer, the following first and second picture respectively shows the tooltip text before and after modification. 
![2](https://user-images.githubusercontent.com/74350258/101432961-c922ea00-3944-11eb-9f43-f083ebce2bd5.png)
![3](https://user-images.githubusercontent.com/74350258/101432962-c9bb8080-3944-11eb-8289-b1cfbc435d47.png)

Reply:
@Geqi-Yan as mentioned, I believe an `ssh` label provider can be created and used instead of directly modifying the `navigator` by checking for scheme. Perhaps others can chime in during their reviews as well but for extensibility purposes I believe a new label provider would be a more appropriate solution.